### PR TITLE
Installation notes.

### PR DESCRIPTION
**Pull request by jesseleite**

### CODE REVIEW

**Comment by jesseleite** [comment]
Tests pass 🙌

File: README.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 Shell stuff will stuff commands into a shell à la `tmux send-keys` or `screen
 stuff`.
 
+## Installation
+
+If your environment is configured to use Python 3 by default:
+```
+pip install shtuff
```
Comment (jfly) [inline README.md]: nit: I think we've been prefixing all shell commands with a `$ `.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
...
+```
+
+Otherwise:
+```
+pip3 install shtuff
```
Comment (jfly) [inline README.md]: nit: I think we've been prefixing all shell commands with a `$ `.

**Review by jfly — COMMENTED** [review]
Thanks! Definitely nice to have this loud and on top of the README =)

**Comment by jesseleite** [comment]
👍 Done!

**Comment by jfly** [comment]
Thanks!!

**Pull request merged**